src/components/Toast/presentors/Buttonless.tsx

```diff
@@ -3,49 +3,63 @@ import styled from '~/modules/theme'
 
 import * as IconFiles from '~/lib/iconFiles'
 import * as Icon from '~/components/Icon'
-
+import * as modules from '~/modules/theme'
 /*
  * Component
  */
 type Props = {
     label: string
+    variant: 'info' | 'warning' | 'progress'
+    color?: string
     text?: string
-    variant: 'info' | 'warning'
+    icon?: IconFileKeys
 }
 
-export const Component = React.memo<Props>(props => {
-    return (
-        <Outer
-            variant={props.variant}
-            data-test={`${props.variant}-buttonless-toast`}
-        >
-            <Icons
-                svg={IconFiles.icons.SingleCheck}
-                size="24px"
-                color={'white'}
-            />
-            {props.text ? (
-                <div>
-                    <Label>{props.label}</Label>
-                    <Text>{props.text}</Text>
-                </div>
-            ) : (
-                <Label>{props.label}</Label>
-            )}
-        </Outer>
-    )
-})
+type IconFileKeys = keyof typeof IconFiles.icons
+
+export const Component = React.memo<Props>(
+    ({
+        label,
+        variant,
+        text,
+        color = modules.defaultTheme.colors.grayScale.S0,
+        icon = IconFiles.icons.SingleCheck
+    }) => {
+        return (
+            <Outer variant={variant} data-test={`${variant}-buttonless-toast`}>
+                {variant === 'progress' ? (
+                    <Icons
+                        svg={IconFiles.icons.Loading}
+                        size="24px"
+                        color={'white'}
+                    />
+                ) : (
+                    <Icons svg={icon} size="24px" color={'white'} />
+                )}
+                {text ? (
+                    <div>
```

Comment:
[IMO]
このdivってstyle持ってないですよね？
React.Fragmentで良い気がします
```suggestion
                    <>
```

Reply:
ありがとうございます!修正しました!